pandas/core/groupby/groupby.py

```diff
@@ -1079,7 +1082,8 @@ def count(self):
         # defined here for API doc
         raise NotImplementedError
 
-    @Substitution(name='groupby', see_also=_common_see_also)
+    @Substitution(name='groupby')
+    @Substitution(see_also=_common_see_also)
```

Comment:
why is that?

Reply:
Within groupby there were 25 SA04 errors which I traced to 4 Substitution decorators.
In these cases the "see_also" had to be subbed first and then "groupby" could be subbed so that the docstring would be built correctly.
These SA04 errors only occurred where there were 2 string parameters within the Substitution decorator. 
When a single Substitution decorator had to substitute 2 parameters it wasn't capable of doing the correct order regardless of the ordering of parameters. Hence I split it into 2 Substitution decorators with one parameter each so the order of substitution would be correct and the docstring could be built correctly.